packages/react/src/components/accordion/Accordion.test.tsx

```diff
@@ -26,4 +30,22 @@ describe('<Accordion /> spec', () => {
     userEvent.click(container.querySelector('[id="accordion-heading"]'));
     expect(container.querySelector('[id="accordion-content"]')).toBeVisible();
   });
+
+  it('should close the accordion when accordion close button is clicked', async () => {
+    jest.useFakeTimers();
+    const { container } = render(
+      <Accordion heading="Foo" id="accordion" initiallyOpen>
+        Bar
+      </Accordion>,
+    );
+
+    await act(async () => {
+      userEvent.click(container.querySelector('[data-testid="accordion-closeButton"]'));
+    });
```

Comment:
It might be possible to omit the act - wrapper around click 🤔

Reply:
- [ ] Remove act wrapper